yolo_model:
  model:
    name: 'yolov3'
  input_shape:
    value: '416'
  confidence_threshold:
    value: 0.2
  batch_size:
    value: 1
  category_num:
    value: 8
